tests_zeeguu_api/test_working_session.py

```diff
@@ -0,0 +1,53 @@
+# coding=utf-8
+
+from unittest import TestCase
+from tests_zeeguu_api.api_test_mixin import APITestMixin
+from tests_zeeguu.rules.user_working_session_rule import WorkingSessionRule
+from datetime import datetime, timedelta
+from random import randint
+
+class UserWorkingSessionTests(APITestMixin, TestCase):
+
+    def setUp(self):
+        super().setUp()
+        working_session = WorkingSessionRule().w_session
+        self.user_id = working_session.user_id
+        self.article_id = working_session.article_id
+        self.is_active = working_session.is_active
+        self.cohort_id = working_session.user.cohort_id
+
+    def test_working_sessions_by_user(self):
+        parameters = dict(
+            user_id = self.user_id,
+            from_date = (datetime.now() - timedelta(minutes=randint(75000, 75000))).isoformat(),
```

Comment:
as far as I can tell `randint(75000, 75000)` is always 75000.

Reply:
@carlospaz7  - is this PR still active?
Then why no answer to this comment?
If not, let's close it... it's getting old :)

Reply:
In the end the development team used their own API and got the data directly from the CORE, also many things have changed since this version (in this version of the API I still called it working session, instead of reading and exercise) I think we can close the PR and only if is needed, I can create the APIs for the reading and exercise sessions.
I totally forgot about it.